extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/config/OidcCommonConfig.java

```diff
@@ -227,27 +227,50 @@ enum Verification {
 
     interface Proxy {
 
+        /**
+         * The name of the proxy configuration to use.
+         * <p>
+         * If a name is configured, it uses the configuration from {@code quarkus.proxy.<name>.*}.
+         * Please note that the 'non-proxy-hosts' option is currently not supported.
+         * If a name is configured, but no proxy configuration is found with that name then an error will be thrown.
+         * <p>
+         * The default proxy configuration is <strong>not</strong> used by default.
```

Comment:
@michalvavrik Can you please clarify this point ?

Reply:
> @michalvavrik Can you please clarify this point ?

That's c&p from TLS registry configuration name we have in the OIDC config. Just because someone configures `quarkus.proxy.host=something` it doesn't mean we are going to use it automatically, they have to tell us which proxy configuration we should use (and if we should use it).

Reply:
>aka we should use quarkus.proxy.some-name.host if someone configured quarkus.oidc.proxy.proxy-configuration-name=some-name.

And if it is not configured, then `quarkus.proxy.host=something` is effective, right ?

Reply:
> > aka we should use quarkus.proxy.some-name.host if someone configured quarkus.oidc.proxy.proxy-configuration-name=some-name.
> 
> And if it is not configured, then `quarkus.proxy.host=something` is effective, right ?

Yes.

Reply:
Oops, sorry, I got it